### PR TITLE
Contact form mobile version

### PR DESCRIPTION
**Pull request by MudasirSherwani**
### Feature work done in this branch.
1. Add Contact Form
2. Add a Footer with Social icons and email address

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -352,4 +351,112 @@
...
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    text-decoration: none;
+  }
+
+  .rectangle-42 {
+    width: 100vh;
```
Comment (leonardodiasb) [inline styles.css]: - [x] You used `vh` value for a `width` attribute when it should be `vw`. Sometimes we implement a lot of classes on CSS and we make some mistakes, but don't forget to double-check typos before submitting a PR.

File: styles.css [hunk 2]
```diff
@@ -352,4 +351,112 @@
...
+    font-weight: 500;
+    font-family: 'Roboto', sans-serif;
+    text-align: center;
+  }
+
+  .contact-form div p {
+    font-size: 16px;
+    line-height: 24px;
+    color: #f5f5ff;
+    font-family: 'Roboto', sans-serif;
```
Comment (leonardodiasb) [inline styles.css]: - [x] Your style of this text matches almost perfectly with the Figma template, but the font-family of the template is different from 'Roboto' for this piece of text. Try changing the font-family to match the Template.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @MudasirSherwani, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of GitHub flow. ✔️
 - Professional PR and README. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (MudasirSherwani) [reply styles.css]: vh unit of width attribute change as vw

**Review by ichala — APPROVED** [review]
Hi @MudasirSherwani ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Great Design✔️
- Form Validations working perfectly ✔️
- Form Submission works perfectly ✔️


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- N/A



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**